Add MVP - SCS infrastructure monitoring blog post (#797)
SovereignCloudStack/website: _i18n/en/_posts/blog/2023-12-06-mvp-monitoring.md

---
layout: post

> title:  "MVP - SCS infrastructure monitoring"

Reviewer: ```suggestion
title:  "SCS observability and monitoring - An opinionated proposal"
```

category: tech
author:
  - "Matej Feder"
avatar:
  - "matofeder.png"
about:
  - "matofeder"
---

> We're excited to announce that our Minimum Viable Product (MVP) for SCS infrastructure
> monitoring has achieved a presentable and viable state. MVP aims to provide a global

Reviewer: ```suggestion
We're excited to announce that our Minimum Viable Product (MVP) for extending the monitoring from the IaaS layer to a solution that covers observability and monitoring full-stack has achieved a presentable and viable state. MVP aims to provide a global
```

> monitoring view of various SCS infrastructure layers, including KaaS, IaaS, and actively

Reviewer: ```suggestion
monitoring view of various SCS layers and components, including KaaS, IaaS, and actively
```

<figure class="figure mx-auto d-block" style="width:50%">
  <a href="{% asset "blog/mvp_monitoring_architecture.png" @path %}">
    {% asset 'blog/mvp_monitoring_architecture.png' class="figure-img w-100" %}
  </a>
</figure>

The SCS Monitoring MVP is developed on the foundation of the [dNation monitoring solution](https://github.com/dNationCloud/kubernetes-monitoring).

## Monitoring of the KaaS Layer

As mentioned earlier, we aim to showcase how the Observer monitors the CSP KaaS offering.
To make this demonstration interactive, we've developed a KaaS mock service, allowing
you to easily create a Kubernetes cluster.

The KaaS mock service is accessible through [Swagger UI](http://213.131.230.7:8080/kaas).

Don't hesitate to [create](http://213.131.230.7:8080/kaas#/Clusters/create_cluster_api_clusters__post) your own Kubernetes cluster
within approximately three minutes. You'll find it displayed on the dedicated [Observer high-level dashboard](http://213.131.230.73:30000/d/kaas-monitoring/kaas-monitoring).
For a deeper exploration of metrics, simply drill down and click on the cluster you created.

> Feel free to experiment, such as triggering monitoring [alerts](http://213.131.230.73:30001/#/alerts) by initiating actions like destroying specific components 😎.

Reviewer: I'm not really happy with this raw reference to the demo setup - since this will likely break within the next 12 months.
I understand this is a nice way of testdriving it - maybe add a disclaimer prepending it that this will only be available in the next couple months?

Author: I agree, - disclaimer added here 1bb2be636490b3ceb13c42a50408312831ca5820
